wrench/src/yaml_frame_writer.rs

```diff
@@ -999,6 +999,9 @@ impl YamlFrameWriter {
 
                     yaml_node(&mut v, "horizontal-offset-bounds", Yaml::Array(horizontal));
                     yaml_node(&mut v, "vertical-offset-bounds", Yaml::Array(vertical));
+
+                    f32_node(&mut v, "applied-offset-x", item.applied_offset.x);
```

Comment:
Thanks for adding wrench support! I think it would be good to add a test for this as well, since there is a risk of breaking such a particular use case.

Reply:
I'm not entirely sure how to do this but I'll ask around.